src/tests/multihost/alltests/test_misc.py

```diff
@@ -19,7 +19,7 @@
 from constants import ds_instance_name
 
 
-@pytest.mark.usefixtures('setup_sssd', 'create_posix_usersgroups'
+@pytest.mark.usefixtures('setup_sssd_gssapi', 'create_posix_usersgroups'
```

Comment:
The gssapi test was removed from the suite, why to set it up?

Reply:
removed this.